src/analysis/thermo/thermoFBA/addLoopLawConstraints.m

```diff
@@ -207,19 +216,19 @@
         -ones(nint,1); % e<
         %-100*ones(nint, 1); % e>
         zeros(linternal,1);];
-
+    

```

Comment:
May you fix these formatting issues? Thanks.

Reply:
Not sure, but this is normal matlab indentation of a block. i.e. if its in a command/end block, and automatically indented, all lines in that lbock are set to a specific level.

Reply:
These additional indentations just appear weirdly here on Github ... When using [atom](https://atom.io/), these indentations are automatically corrected.

Reply:
Well, they are what matlab does for automatic indentation (and I actually think it makes sense, if you have empty lines to keep them on the same indentation level. 
I can remove all the empty lines, but removing the indentation seems odd to me.